content/javascript/concepts/strings/strings.md

```diff
@@ -43,3 +43,19 @@ let age = 7;
 // String interpolation
 `Tommy is ${age} years old.`;
 ```
+
+## Escaping Characters
+
+On occasion in may be necessary to include characters in a string that may have reserved meanings or to apply additional formatting. Characters, such as quotes, can be included by prepending a `\`. And there are a number of predefined escape sequences such as `\n` or `\t` to add the formatting such as line break or tab respectively.
```

Comment:
```suggestion
On occasion it may be necessary to include characters in a string that may have reserved meanings or to apply additional formatting. Characters, such as quotes, can be included by prepending a `\`. There are a number of predefined escape sequences such as `\n` or `\t` to add the formatting such as line break or tab respectively.
```